build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.1'
```

Comment:
Don't change configuration files.
If you are required to change, exclude them from the commit.